cmd/skopeo/proxy.go

```diff
@@ -166,6 +167,11 @@ type proxyHandler struct {
 	activePipes map[uint32]*activePipe
 }
 
+type layersInfo struct {
+  Success string 
+  Layers []types.BlobInfo
```

Comment:
Consider not exporting all of the `BlobInfo` — The “operation/algorithm” fields were really a mistake.